spec/restaurant_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'restaurant.rb'
+require 'menu.rb'
+
+describe Restaurant do
+  subject { 
+    menu = double(Menu.new)
```

Comment:
I may be incorrect, but I was under the impression that we were not supposed to rely on the other classes during testing. In this case, you are referring directly to the Menu class from the restaurant class.